internal/pkg/scorecard/test_definitions.go

```diff
@@ -123,29 +123,32 @@ func MergeSuites(suites []TestSuite) ([]TestSuite, error) {
 	mergedSuites := []TestSuite{}
 	for _, suiteSlice := range suiteMap {
 		testMap := make(map[string][]TestResult)
+		logs := ""
```

Comment:
This is probably a good opportunity to use [`strings.Builder`](https://golang.org/pkg/strings/#Builder)